x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/connectors/create_connector/configuration_step.tsx

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useEffect } from 'react';
+
+import { useValues } from 'kea';
+
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiPanel,
+  EuiSpacer,
+  EuiTitle,
+  EuiText,
+  EuiButton,
+  EuiButtonEmpty,
+  EuiProgress,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+// import { FormattedMessage } from '@kbn/i18n-react';
+import { ConnectorConfigurationComponent, ConnectorStatus } from '@kbn/search-connectors';
+
+import { ConnectorViewLogic } from '../../connector_detail/connector_view_logic';
+
+interface ConfigurationStepProps {
+  currentStep: number;
+  isNextStepEnabled: boolean;
+  setCurrentStep: Function;
+  setNextStepEnabled: Function;
+  setSyncing: Function;
+  syncing: boolean;
+  title: string;
+}
+
+export const ConfigurationStep: React.FC<ConfigurationStepProps> = ({
+  title,
+  currentStep,
+  setCurrentStep,
+  isNextStepEnabled,
+  setNextStepEnabled,
+  setSyncing,
+  syncing,
+}) => {
+  const { connector } = useValues(ConnectorViewLogic);
+  if (connector) {
+    connector.status = 'created' as ConnectorStatus;
```

Comment:
Note to future reviewers: This was for the demo, should go away once the real data is used. 
DON'T MERGE WITH THIS STILL ON.